single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -249,7 +249,8 @@ protected void drawVoltageLevel(String prefixId,
         drawSnakeLines(prefixId, root, graph, metadata, styleProvider, anchorPointProvider);
 
         // Drawing the nodes outside the voltageLevel graphs (multi-terminal nodes)
-        drawNodes(prefixId, root, graph, metadata, initProvider, styleProvider, graph.getMultiTermNodes());
+        drawNodes(prefixId, root, graph, metadata, initProvider, styleProvider,
```

Comment:
you'd better change the signature of drawNodes (taking a `Collection<? extends Node>` instead of `Collection<Node>`) to avoid creating a new List

Reply:
thanks I was mad with that ugly trick